world/clientlist.cpp

```diff
@@ -100,114 +100,156 @@ ClientListEntry* ClientList::GetCLE(uint32 iID) {
-								countCLEIPs->SetOnline(CLE_Status::Offline); // Remove the connection
+								LogClientLogin("Disconnect: Account [{}] on IP [{}]", cle->LSName(), ip_string);
+								cle->SetOnline(CLE_Status::Offline); // Remove the connection
 								iterator.RemoveCurrent();
 								continue;
 							}
 						}
-					} else if ((countCLEIPs->Admin() < RuleI(World, AddMaxClientsStatus)) || (RuleI(World, AddMaxClientsStatus) < 0)) { // Else if the Admin status of the connection is not eligible for the higher limit, or there is no higher limit (AddMaxClientStatus < 0)
-						if(RuleB(World, IPLimitDisconnectAll)) {
-							LogClientLogin("Disconnect: All accounts on IP [{}]", long2ip(countCLEIPs->GetIP()).c_str());
-							DisconnectByIP(iIP);
+					} else if (
+						cle->Admin() < RuleI(World, AddMaxClientsStatus) ||
+						RuleI(World, AddMaxClientsStatus) < 0
+					) { // Else if the Admin status of the connection is not eligible for the higher limit, or there is no higher limit (AddMaxClientStatus < 0)
+						if (RuleB(World, IPLimitDisconnectAll)) {
+							LogClientLogin("Disconnect: All Accounts on IP [{}]", ip_string);
+							DisconnectByIP(in_ip);
 							return;
 						} else {
-							LogClientLogin("Disconnect: Account [{}] on IP [{}]", countCLEIPs->LSName(), long2ip(countCLEIPs->GetIP()).c_str());
-							countCLEIPs->SetOnline(CLE_Status::Offline); // Remove the connection
+							LogClientLogin("Disconnect: Account [{}] on IP [{}]", cle->LSName(), ip_string);
+							cle->SetOnline(CLE_Status::Offline); // Remove the connection
 							iterator.RemoveCurrent();
 							continue;
 						}
-					} else if (IPInstances > RuleI(World, AddMaxClientsPerIP)) { // else they are eligible for the higher limit, but if they exceed that
-						if(RuleB(World, IPLimitDisconnectAll)) {
-							LogClientLogin("Disconnect: All accounts on IP [{}]", long2ip(countCLEIPs->GetIP()).c_str());
-							DisconnectByIP(iIP);
+					} else if (
+						RuleI(World, AddMaxClientsPerIP) > 0 &&
+						count > RuleI(World, AddMaxClientsPerIP)
+					) { // else they are eligible for the higher limit, but if they exceed that
+						if (RuleB(World, IPLimitDisconnectAll)) {
+							LogClientLogin("Disconnect: All Accounts on IP [{}]", ip_string);
+							DisconnectByIP(in_ip);
 							return;
 						} else {
-							LogClientLogin("Disconnect: Account [{}] on IP [{}]", countCLEIPs->LSName(), long2ip(countCLEIPs->GetIP()).c_str());
-							countCLEIPs->SetOnline(CLE_Status::Offline); // Remove the connection
+							LogClientLogin("Disconnect: Account [{}] on IP [{}]", cle->LSName(), ip_string);
+							cle->SetOnline(CLE_Status::Offline); // Remove the connection
 							iterator.RemoveCurrent();
 							continue;
 						}
 					}
 				}
 			}
 		}
+
 		iterator.Advance();
 	}
 }
 
-uint32 ClientList::GetCLEIPCount(uint32 iIP) {
-	ClientListEntry* countCLEIPs = 0;
+uint32 ClientList::GetCLEIPCount(uint32 in_ip) {
+	ClientListEntry* cle = 0;
```

Comment:
Oops, had another tab open for some reason with this highlighted. `= nullptr` here